tasks/windows.ps1

```diff
@@ -0,0 +1,46 @@
+[CmdletBinding()]
+param(
+  # NOTE: init.json cannot yet be shared, so must have windows.json / windows.ps1
+  [Parameter(Mandatory = $true)]
+  [String]
+  $Name,
+
+  [Parameter(Mandatory = $true)]
+  [ValidateSet('start', 'stop', 'restart', 'status')]
+  [String]
+  $Action,
+
+  [Parameter(Mandatory = $false)]
+  [ValidateSet('automatic', 'manual', 'disabled')]
+  [String]
+  $StartType
+)
+
+$ErrorActionPreference = 'Stop'
+
+$service = Get-Service -Name $Name
+
+switch ($StartType.ToLower())
```

Comment:
powershell switch statement is case-insensitive by default, so tolower (while stating intent clearly) is redundant

Reply:
if tolower is kept, consider tolowerinvariant as that won't change user input

Reply:
Thanks - forgot about that! Also meant to put a TODO there to double check how the Puppet type system metadata works... The `[Enum]` may already be case sensitive, in which case some of the validation here is moot.